src/main/webapp/dev/js/common/instructorFeedbackResults.js

```diff
@@ -149,19 +149,46 @@ function replaceButtonHtmlAndTooltipText(button, from, to) {
 function expandOrCollapsePanels(expandCollapseButton, panels) {
     const STRING_EXPAND = 'Expand';
     const STRING_COLLAPSE = 'Collapse';
+    const expandCollapseStudentsButton = 'a[id^="collapse-panels-button-section-"]';
+    const expandCollapseTeamButton = 'a[id^="collapse-panels-button-team-"]';
+    const isButtonInExpandMode = $(expandCollapseButton).html().trim().startsWith(STRING_EXPAND);
+    const isCollapsePanelsButton = !panels;
+    /*
+     * When collapsing expandCollapseButton, expandCollapseTeamButton and expandCollapseStudentButton
+     * should be in collapsed state, so that next time when we expand the main panel, we find
+     * the expandCollapseTeamButton and expandCollapseStudentButton in collapsed state.
+     */
+    if (isCollapsePanelsButton && !isButtonInExpandMode) {
+        if ($(expandCollapseStudentsButton).length
+            && $(expandCollapseStudentsButton).html().trim().startsWith(STRING_EXPAND)) {
```

Comment:
Can we extract out a method `isInExpandMode(button)`? We seem to be calling `.html().trim().startsWith(` a lot.